test/conformance/kuttl/verifyImages/clusterpolicy/standard/noconfigmap-diffimage-success/policy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  annotations:
+    pod-policies.kyverno.io/autogen-controllers: none
+  name: image-verify-polset
+spec:
+  background: false
+  failurePolicy: Fail
+  rules:
+  - context:
+    - configMap:
+        name: myconfigmap
+        namespace: mynamespace
+      name: myconfigmap
+    match:
+      resources:
```

Comment:
All new rules need to use `any` or `all` under `match`/`exclude` blocks.

Reply:
fixed